packages/sit-onyx/src/components/OnyxTooltip/OnyxTooltip.vue

```diff
@@ -68,6 +83,18 @@ const type = computed(() => {
   if (typeof props.open === "string") return props.open;
   return "hover";
 });
+// classes for the tooltip | computed to prevent bugs
+const tooltipClasses = computed(() => {
+  return {
+    "onyx-tooltip--danger": props.color === "danger",
+    "onyx-tooltip--fit-parent": props.fitParent,
+    "onyx-tooltip--hidden": !isVisible.value,
+    [`onyx-tooltip--${openDirection.value}`]: props.position === "auto",
+    [`onyx-tooltip--${props.position}`]: props.position !== "auto",
+    [`onyx-tooltip--align--${wedgePosition.value}`]: props.align === "auto",
+    [`onyx-tooltip--align--${props.align}`]: props.align !== "auto",
```

Comment:
```suggestion
    [`onyx-tooltip--${wedgePosition.value}`]: props.align === "auto",
    [`onyx-tooltip--${props.align}`]: props.align !== "auto",
```

in BEM there is only 1 `--` allowed, so we need to rename it e.g. like this